src/controllers/managing.officer.controller.ts

```diff
@@ -7,24 +7,35 @@ import {
   removeManagingOfficer
 } from "../utils/managing.officer.individual";
 
-import { BENEFICIAL_OWNER_TYPE_URL, MANAGING_OFFICER_PAGE } from "../config";
+import * as config from "../config";
+
+import { isActiveFeature } from "../utils/feature.flag";
+import { getUrlWithParamsToPath } from "../utils/url";
 
 export const get = (req: Request, res: Response) => {
-  getManagingOfficer(req, res, BENEFICIAL_OWNER_TYPE_URL, MANAGING_OFFICER_PAGE);
+  getManagingOfficer(req, res, config.BENEFICIAL_OWNER_TYPE_URL, config.MANAGING_OFFICER_PAGE);
 };
 
 export const getById = (req: Request, res: Response, next: NextFunction) => {
-  getManagingOfficerById(req, res, next, BENEFICIAL_OWNER_TYPE_URL, MANAGING_OFFICER_PAGE);
+  getManagingOfficerById(req, res, next, config.BENEFICIAL_OWNER_TYPE_URL, config.MANAGING_OFFICER_PAGE);
 };
 
 export const post = (req: Request, res: Response, next: NextFunction) => {
-  postManagingOfficer(req, res, next, BENEFICIAL_OWNER_TYPE_URL, true);
+  postManagingOfficer(req, res, next, getNextPageURL(req), true);
 };
 
 export const update = (req: Request, res: Response, next: NextFunction) => {
-  updateManagingOfficer(req, res, next, BENEFICIAL_OWNER_TYPE_URL, true);
+  updateManagingOfficer(req, res, next, getNextPageURL(req), true);
 };
 
 export const remove = (req: Request, res: Response, next: NextFunction) => {
-  removeManagingOfficer(req, res, next, BENEFICIAL_OWNER_TYPE_URL, true);
+  removeManagingOfficer(req, res, next, getNextPageURL(req), true);
+};
+
+const getNextPageURL = (req: Request): string => {
```

Comment:
How about changing the name to match what @markpit suggested and implemented in BO Other and I adopted in BO Gov.